### PR TITLE
Add a test for the mosquitto example.

### PR DESCRIPTION
**Pull request by elopio**
LP: 1564181

### CODE REVIEW

**Comment by elopio** [comment]
retest this please

**Comment by elopio** [comment]
retest this please

File: examples/mosquitto/subscribe.py [hunk 1]
```diff
@@ -14,14 +15,22 @@
...
     # Ignore the unused arguments.
     del unused1, unused2
-    print(message.topic + ' ' + str(message.payload))
+    _log(message.topic + ' ' + str(message.payload))
+
+
+def _log(message):
+    print(message)
+    log_file_path = os.path.join(
+        os.getenv('SNAP_APP_USER_DATA_PATH'), 'mosquitto.subscriber.log')
```
Comment (sergiusens) [inline examples/mosquitto/subscribe.py]: This should be `SNAP_USER_DATA`

**Comment by elopio** [comment]
retest this please

File: examples/mosquitto/subscribe.py (hunk 1 above)
Comment (elopio) [inline examples/mosquitto/subscribe.py]: are you sure? I get a permission error trying to write to SNAP_USER_DATA.

**Comment by elopio** [comment]
retest this please

File: examples/mosquitto/subscribe.py (hunk 1 above)
Comment (sergiusens) [inline examples/mosquitto/subscribe.py]: `SNAP_APP_USER_DATA_PATH` shouldn't exist anymore :confused:

**Comment by elopio** [comment]
this requires a new rule in the firewall: https://portal.admin.canonical.com/90236
Closing until that's done.

**Pull request closed without merging**